exercises/practice/roman-numerals/roman-numerals_test.sql

```diff
@@ -0,0 +1,44 @@
+-- Setup test table and read in student solution:
+.read ./test_setup.sql
+
+-- Test cases:
+INSERT INTO tests (name, uuid,
+                    number, expected)
```

Comment:
Is this indentation intentional?

Reply:
It's consistent, and if we want to change it, I think it should be a separate PR.